lms/views/onedrive.py

```diff
@@ -21,8 +24,19 @@ def redirect_uri(_request):
     renderer="json",
 )
 def verify_domain(request):
-    return {
-        "associatedApplications": [
-            {"applicationId": request.registry.settings["onedrive_client_id"]}
-        ]
-    }
+    application_id = request.registry.settings["onedrive_client_id"]
+    response_text = textwrap.dedent(
```

Comment:
TIL about texwrap.dedent but I think doing this on a template and getting the size back for `content_length` would have been more verbose.

Reply:
Did you consider doing it manually?

```python
def verify_domain(request):
    application_id = request.registry.settings["onedrive_client_id"]
    response_text = f"""{{
  "associatedApplications": [
    {{
      "applicationId": "{application_id}"
    }}
  ]
}}"""
    request.response.content_length = len(response_text)

```

Too ugly?

Reply:
Yeah, tried that but it was not very nice.